### DISCUSSION TITLE
Interstellar is buggy

### DISCUSSION THREAD

**Discussion by ghost** [UseInterstellar/Interstellar#630] [General]
From, Koyeb to Codespaces, I did everything possible and I'm getting annoyed by the popup of every search result after deployment, "RangeError: Failed to construct 'Response': The status provided (0) is outside the range [200, 599]." It always shows up and never gives me a reliable amount of downtime on interstellar and it usually works, codespaces is not reliable and i did npm i && npm run start so idk what to do now

**Comment by replitidk** [comment]
@S3CR3TAS1AN this may belong in issues i dont know why it is doing this but what i would try is deleting the code space on github make a new code space and put pnpm i && pnpm start rather than npm i && npm start i all ways use the pnpm i && pnpm start and it works if this does not work put it in issues for xbubbo to deal with

**Comment by replitidk** [comment]
this should not be happening in the first place but i think you may just need to do pnpm i && pnpm start maybe i am not sure

**Comment by replitidk** [comment]
also on github codespaces are you clicking make public since it says to in the readme file that you need to click make public
